Presentation.Web/app/components/it-advice/It-advice.controller.ts

```diff
@@ -7,6 +7,7 @@
                 $scope.type = type;
                 $scope.object = object;
                 $scope.advicename = advicename;
+                $scope.hasWriteAccess = hasWriteAccess;
```

Comment:
Denne del var fjernet i forhold til master udgaven. Grunden til at det virkede på it-system og it-contract er fordi den controller der bruges når man åbner de enkelte entities endnu ikke er blevet ændret til typescript controllers som på it projekt og DBR. De oprindelige controllers sætter nemlig hasWriteAccess på scope mens de nye sætter dem som værdi på controlleren.